### PR TITLE
Update vulnerable dependencies

### PR DESCRIPTION
**Pull request by malwoodsantoro**
Updates `randomatic` and `cached-path-relative` to latest versions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,9 @@
   },
   "dependencies": {
     "@turf/bbox": "^6.0.1",
+    "cached-path-relative": "^1.0.2",
     "lodash": "^4.17.10",
+    "randomatic": "^3.1.1",
```
Comment (andrewharvey) [inline package.json]: are these direct dependencies or pulled in from an upstream library?

Comment (malwoodsantoro) [reply package.json]: Looks like they are pulled in from an upstream library @andrewharvey is there a better way to update them?

Comment (andrewharvey) [reply package.json]: I think (but I'm not expert) `npm audit fix --force` is the proper way to do it (apart from getting the upstream libraries to fix their dependencies).

This way it only changes the `package-lock.json`.

Comment (malwoodsantoro) [reply package.json]: @andrewharvey thanks! That makes sense. I reverted my changes and tried `npm audit fix --force` but nothing changed in my `package-lock.json`. Any ideas?

Comment (malwoodsantoro) [reply package.json]: I think update just budo should do the trick @andrewharvey 👍

Comment (andrewharvey) [reply package.json]: :+1: Yes that did it for me, in combination with `npm audit fix`. Then `npm audit` says there are no more issues. You can drop randomatic and cached-path-relative from the direct dependencies now.

**Comment by malwoodsantoro** [comment]
Should be good to go now, @andrewharvey! Thanks.

**Pull request merged**